sql/postgresql/pggrammar/pggrammar.sandbox/App.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please remove files not related to grammar, examples, or tests scripts (seems like all files from `sql/postresql/pggrammar/pggrammar.sandbox`).